lib/sentence-generator/index.js

```diff
@@ -235,7 +261,7 @@ class SentenceGenerator extends events.EventEmitter {
                         {}
                     )]);
                 }
-                const pcanonical = ex.value.schema.getArgument(pname).canonical || clean(pname);
+                const pcanonical = ex.value.schema.getArgCanonical(pname);
```

Comment:
Is this new in ThingTalk? And do you need to bump the thingtalk dep to pass Travis?